ui/tests/integration/components/config-ui/messages/page/list-test.js

```diff
@@ -96,4 +95,44 @@ module('Integration | Component | messages/page/list', function (hooks) {
       assert.dom(`[data-linked-block-title="${message.id}"]`).hasText(message.title);
     }
   });
+
+  test('it should show max message warning modal', async function (assert) {
```

Comment:
This test is causing pagination accessibility errors. Open to thoughts on how to refine this test so it doesn't cause these accessibility errors. I checked in the ui with the axedevtools and didn't see any accessibility errors, so I think it's how I set up the meta tag here.

Reply:
Edit: I was able to avoid these accessibility errors when I see pageSize to 100. I noticed that accessibility errors don't occur for disabled prev / next buttons.